packages/runtime/src/next/components/tests/global-element-rendering.test.tsx

```diff
@@ -224,9 +225,6 @@ describe('Page', () => {
   test('correctly renders a localized global element', async () => {
     await testGlobalElementRendering({
       locale: 'fr-FR',
-      localizedResourcesMap: {
-        [globalElementId]: localizedGlobalElementId,
-      },
```

Comment:
Simple presence of `localizedResourcesMap` in the cache no longer triggers global component localization, moved the cache inside of the `testGlobalElementRendering` helper.